spritzle/resource/config.py

```diff
@@ -20,10 +20,28 @@
 #   Boston, MA    02110-1301, USA.
 #
 
+from aiohttp import web
 
+routes = web.RouteTableDef()
+
+
+@routes.get('/config')
 async def get_config(request):
-    raise NotImplementedError
+    config = request.app['spritzle.config']
+    return web.json_response(dict(config))
+
+
+@routes.put('/config')
+async def put_config(request):
+    config = request.app['spritzle.config']
+    new_values = await request.json()
+    config.data = new_values
```

Comment:
I think you should call config.save() after setting the data.

Reply:
Good call.